example-site/components/FormExampleMultiStep/FormExampleMultiStepActions.tsx

```diff
@@ -24,7 +24,7 @@ export const FormExampleMultiStepActions = () => {
 				<FormDivider />
 				<ButtonGroup>
 					<Button type="submit" variant="primary" loading={isSubmittingStep}>
-						{currentStep === 4 ? 'Submit form' : 'Continue'}
```

Comment:
This should stay as `Submit form` as it's not nested inside al ist of tasks

Reply:
good ole' global search